plugins/qtosgrave/qtosgviewer.cpp

```diff
@@ -1441,7 +1441,7 @@ RaveTransform<float> QtOSGViewer::GetCameraTransform() const
     boost::mutex::scoped_lock lock(_mutexGUIFunctions);
```

Comment:
Probably no need for this lock?

Reply:
other gui functions could change the camera transformation .. ?